Sources/BraintreeDropIn/BTDropInController.m

```diff
@@ -268,9 +268,6 @@ - (void)threeDSecureVerification:(BTPaymentMethodNonce *)tokenizedCard {
     paymentFlowDriver.viewControllerPresentingDelegate = self;
 
     BTThreeDSecureRequest *request = self.dropInRequest.threeDSecureRequest;
-    if (!request) {
-        request = [[BTThreeDSecureRequest alloc] init];
-    }
```

Comment:
I was thinking - this PR should also remove wherever the logic was that set the amount on the `ThreeDSecureRequest` from the `BTDropInResult` ... however after searching the SDK I don't see this happening anywhere. So this looks sufficient, then!

Reply:
Yeah, I was looking for that too! I think we must have removed `amount` previously.